docs/3.1/versions.json

```diff
@@ -1,6 +1,6 @@
 {
-  "daml": "2.8.0-snapshot.20231118.12382.0.v86cb8054",
-  "canton": "2.9.0-snapshot.20240402.11920.0.v2507c677",
+  "daml": "3.1.0-snapshot.20240416.13009.0.v4f422ab2",
+  "canton": "3.1.0-snapshot.20240416.13095.0.v4056c27a",
```

Comment:
You'll probably want to try this as a separate PR.

Reply:
Thank you, let me try that.